src/main/menu.ts

```diff
@@ -60,6 +60,10 @@ export default class MenuBuilder {
             accelerator: 'CommandOrControl+S',
             click: () => takeScreenshot(this.menubar),
           },
+          {
+            label: 'View Application Logs',
```

Comment:
Confirming, no need for an accelerator shortcut? `CommandOrControl+L` 🤷‍♂️

Reply:
Figured it's not a commonly used action, so didn't keep a shortcut 😅